plugins-list.yaml

```diff
@@ -16,31 +17,37 @@ plugins/auth-backend-module-guest-provider:
 plugins/auth-backend-module-microsoft-provider:
 plugins/auth-backend-module-oauth2-provider:
 plugins/auth-backend-module-oauth2-proxy-provider:
-plugins/auth-backend-module-oidc-provider:
+# plugins/auth-backend-module-oidc-provider: # Included as static plugin in RHDH
```

Comment:
Why is the text different here ? Compare with `Added as a static plugin in RHDH`